storage/object/api-cli/create-bucket-policy.mdx

```diff
@@ -0,0 +1,258 @@
+---
+meta:
+  title: Creating and applying a bucket policy
+  description: This page shows how to create and apply a bucket policy to a specific bucket using the Scaleway CLI
+content:
+  h1: Creating and applying a bucket policy
+  paragraph: This page shows how to create and apply a bucket policy to a specific bucket using the Scaleway CLI
+tags: bucket policy bucket bucket-policy object storage object-storage s3
+dates:
+  validation: 2023-10-18
+  posted: 2023-09-29
+categories:
+  - storage
+  - object-storage
+---
+
+<Macro id="storage-bucket-policy" />
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created an Object Storage bucket](/storage/object/how-to/create-a-bucket/)
+  - You have [installed the AWS CLI](/storage/object/api-cli/object-storage-aws-cli/)
+  - You have [created an IAM policy](/identity-and-access-management/iam/how-to/create-policy/) to grant access to your users and applications
+</Message>
+
+## How to create a bucket policy
+
+1. Create a `bucket-policy.json` file.
+
+2. Open it in a code editor and paste the code below inside. This statement denies access to everyone **EXCEPT** the specified user.
+    ```json
+    {
+        "Version": "2023-06-07-alpha",
+        "Id": "MyBucketPolicy",
+        "Statement": [
+            {
+                "Sid": "Deny access to all except user",
+                "Effect": "Deny",
+                "NotPrincipal": {
+                    "SCW": "user_id:<USER_ID>"
+                },
+                "Action": "*",
+                "Resource": [
+                    "<BUCKET_NAME>",
+                    "<BUCKET_NAME>/*"
+                ]
+            }
+        ]
+    }
+    ```
+3. Add a second statement using the `Principal` and `NotAction` elements. This statement denies every action **EXCEPT** the ones specified.
+    ```json
+    {
+        "Sid": "Deny all actions except GetObject and ListBucket",
+        "Effect": "Deny",
+        "Principal": {
+            "SCW": "user_id:<USER_ID>"
+        },
+        "NotAction": [
+            "s3:GetObject",
+            "s3:ListBucket"
+        ],
+        "Resource": [
+            "<BUCKET_NAME>",
+            "<BUCKET_NAME>/*"
+        ]
+    }
```

Comment:
Is this second statement necessary? IMO it makes the experience more complex

Reply:
Plus if we are talking to unexperienced users, they will not know where to put the second statement in the json